src/components/Notifications/EventLog/EventLogToolbar.tsx

```diff
@@ -61,6 +61,22 @@ const actionStatusMetadata = [
         chipValue: 'Success',
         label: <span><CheckCircleIcon color='green' /> Success</span>
     },
+    {
+        value: 'false',
+        chipValue: 'Sent',
+        label: <span><CheckCircleIcon color='green' /> Sent</span>
+    },
+    {
+        value: 'false',
+        chipValue: 'Warning',
+        label: <span><ExclamationTriangleIcon color='#F0AB00' /> Warning </span>
```

Comment:
Might be good if we could use CSS var `--pf-chart-donut--threshold--warning--Color` and custom class I am just not sure if it's applicable to icons.